samples/embedded-login/.env.example

```diff
@@ -1,7 +1,6 @@
 AM_URL=$AM_URL
-CLIENT_ID=$CLIENT_ID
+WEB_OAUTH_CLIENT=$WEB_OAUTH_CLIENT
```

Comment:
I'd like to keep these alphabetized, if possible. It just makes it easier to find variables when the list grows.

Reply:
Good call, will fix.